### PR TITLE
Add my pin to the map! YAY!

### PR DESCRIPTION
**Pull request by kristychapman2016**
Description needed here?

### CODE REVIEW

**Comment by Rmondragon1987** [comment]
what happened?

**Comment by ktlnbstn** [comment]
to pass tests you need to add your latitude and longitude information so the bot can add you.
they also asked that you tag the teachers and one ore step...

File: _pins/kristychapman2016.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle:kristychapman2016
```
Comment (crichID) [inline _pins/kristychapman2016.json]: @kristychapman2016 you will need to have a space after each `:` in order for the checks to pass! If you need help, please check out this video: https://vimeo.com/201011288

**Comment by hollenberry** [comment]
@kristychapman2016  I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6568/commits/355dba58550ada7c40a26b4ed097c28b06a6beef. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26, I will go ahead and do it just to keep the repository clean!

**Pull request merged**